test/UniversalRewardsDistributorTest.sol

```diff
@@ -374,22 +374,36 @@ contract UniversalRewardsDistributorTest is Test {
 
         vm.prank(owner);
         vm.expectEmit(address(distributionWithTimeLock));
-        emit EventsLib.PendingRootSet(0, 0, 0);
+        emit EventsLib.PendingRootRevoked();
         distributionWithTimeLock.revokePendingRoot();
 
         PendingRoot memory pendingRoot = _getPendingRoot(distributionWithTimeLock);
         assertEq(pendingRoot.root, bytes32(0));
         assertEq(pendingRoot.submittedAt, 0);
     }
 
-    function testRevokePendingRootShouldRevertIfNotOwner(bytes32 proposedRoot, address caller) public {
-        vm.assume(proposedRoot != bytes32(0) && caller != owner);
+    function testRevokePendingRootShouldRevokeWhenCalledWithUpdater() public {
```

Comment:
I am not blocking for this but here is my reasoning:
- a hard requirement would be to have it start with the functions name in camelCase: `testRevokePendingRoot` so we can filter tests by functions (`--match-test RevokePendingRoot`), which is easy when applying changes to a single function instead of running the whole test suite
- a soft requirement would be to have it the shortest yet descriptive enough to avoid clashing ; only a soft requirement because these are tests and we don't really care about naming conventions if it's not useful to something?